packages/lesswrong/components/collections/CollectionsCard.tsx

```diff
@@ -106,7 +106,7 @@ const CollectionsCard = ({ collection, url, mergeTitle=false, classes }: {
         </Typography>
       </div>
       <div className={classes.media}>
-        <CloudinaryImage publicId={collection.imageId} />
+        <CloudinaryImage publicId={collection.imageId} width={307} height={86} />
```

Comment:
Why did you need to add dimensions to these?

Reply:
LessWrong apparently resized their images before uploading to cloudinary. Fortunately, cloudinary allows resizing on retrieval, so I just did that.